Sources/HttpParser.swift

```diff
@@ -37,10 +37,11 @@ public class HttpParser {
             return []
         }
         let queryStart = url.index(after: questionMark)
-        guard url.endIndex > queryStart else {
-            return []
-        }
-        let query = String(url[queryStart..<url.endIndex])
+
+        guard url.endIndex > queryStart,
+            let query = String(url[queryStart..<url.endIndex])
```

Comment:
This already returned a non-optional string. This change shouldn't even compile as far as I can tell.

Reply:
That works up to Swift 3, it turns out. Swift 4 does enforce conditional binging in guards.